spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/couchbase/CouchbaseDataProperties.java

```diff
@@ -39,6 +40,11 @@
 	 */
 	private Consistency consistency = Consistency.READ_YOUR_OWN_WRITES;
 
+	/**
+	 * The type key to use if a complex type was identified.
+	 */
```

Comment:
I think it would be good if this description was similar to the javadoc for `typeKey()`. Perhaps something like the following:

> Name of the field that will store the type information for complex types when using MappingCouchbaseConverter